registration/app/registration/services/azure/NotificationsHubRegistrar.scala

```diff
@@ -23,29 +23,37 @@ class NotificationHubRegistrar(
   override val providerIdentifier = "azure"
   val logger = Logger(classOf[NotificationHubRegistrar])
 
-  override def register(lastKnownChannelUri: String, registration: Registration): RegistrarResponse[RegistrationResponse] = {
-    findRegistrationResponses(lastKnownChannelUri).flatMap {
+  override def register(pushToken: String, registration: Registration): RegistrarResponse[RegistrationResponse] = {
+    findRegistrationResponses(pushToken).flatMap {
       case Right(Nil) => createRegistration(registration)
       case Right(azureRegistration :: Nil) => updateRegistration(azureRegistration, registration)
```

Comment:
Is it worth renaming this now?

Reply:
Could be separate but it got really itchy :)